### PR TITLE
chore: adding question on accessibility

### PR DESCRIPTION
**Pull request by HemanthTigga**
## Summary of changes

1. Added a new quiz question to the codebase.
2. The new question focuses on web accessibility and the importance of the 'alt' attribute in HTML image tags.
3. Included four answer choices, one correct answer, and three distractors.
4. Provided explanations for both the correct answer and the distractors to help users understand the concepts better.
5. Added a link to an external resource for further reference on web accessibility.

These changes enhance the codebase by incorporating a new educational element related to web accessibility and image 'alt' attributes in HTML.

## Checklist

- [ x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x ] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x ] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x ] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [ x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

**Comment by HemanthTigga** [comment]
please merge the PR

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const accessibilityQuiz = [
+  {
+    Question: "Why is it important to add alternative text to images on web pages?",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:3]: we already have a similar question on line 97

please choose another accessibility question to add instead

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/accessibility-quiz.ts (hunk 1 above)
Comment (HemanthTigga) [reply src/data/accessibility-quiz.ts:3]: Thanks for letting me know, I will change the question.👍

**Review by HemanthTigga — COMMENTED** [review]
i've changed the question please review it.

File: src/data/accessibility-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,13 @@
 const accessibilityQuiz = [
+  {
+    Question: "What legal repercussions can businesses face for failing to meet website accessibility requirements?",
+    Answer: "Businesses that fail to meet website accessibility requirements may face lawsuits, financial penalties, and reputational damage, in addition to the potential loss of customers and legal compliance obligations.",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:4]: the answer is to long.
remember that is needs to fit in the answer container on the site
https://developerquiz.org/

```suggestion
    Answer: "Businesses can face lawsuits or financial penalties",
```

File: src/data/accessibility-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,13 @@
...
+    Distractor1: "What are the top five e-commerce trends for 2023?",
+    Distractor2: "How can businesses optimize their supply chain management processes?",
+    Distractor3: "What are the benefits of adopting cloud computing in the corporate sector?",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:7]: we shouldn't use questions for the distractors.

```suggestion
    Distractor1: "no repercussions",
    Distractor2: "they will serve 10 years in jail",
    Distractor3: "they will do 2500 hours of community service",
```

File: src/data/accessibility-quiz.ts [hunk 4]
```diff
@@ -1,4 +1,13 @@
...
+    Explanation: "This question highlights the legal consequences businesses may encounter when neglecting website accessibility, emphasizing the importance of compliance in the digital age.",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:8]:
```suggestion
    Explanation: "Businesses that fail to meet website accessibility requirements may face lawsuits, financial penalties, and damage to their reputation. Maintaining good accessibility standards is good for everyone in this digital age.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Please review the following changes

**Review by HemanthTigga — COMMENTED** [review]
changes done

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**